x/debug.go

```diff
@@ -27,8 +27,24 @@ import (
 	"github.com/dgraph-io/badger/v4"
 	bpb "github.com/dgraph-io/badger/v4/pb"
 	"github.com/dgraph-io/dgraph/protos/pb"
+	"github.com/golang/glog"
 )
 
+func PrintRollup(plist *pb.PostingList, parts map[uint64]*pb.PostingList, baseKey []byte, ts uint64) {
+	k, _ := Parse(baseKey)
+	glog.V(2).Infof("[TXNLOG] DOING ROLLUP for key: %+v at timestamp: %v\n", k, ts)
+}
+
+func PrintMutationEdge(plist *pb.DirectedEdge, key ParsedKey, startTs uint64) {
+	glog.V(2).Infof("[TXNLOG] ADDING MUTATION at TS: %v, key: %v, value: %v \n", startTs, key.String(), plist)
+}
+
+func PrintOracleDelta(delta *pb.OracleDelta) {
+	for _, status := range delta.Txns {
+		glog.V(2).Infof("[TXNLOG] COMMITING: startTs: %v, commitTs: %v\n", status.StartTs, status.CommitTs)
```

Comment:
is it possible to print just one log line for one txn?
and remove the \n at the end